src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import './CreateCourse.css';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import Textarea from '../../common/Textarea/Textarea';
+import formatDuration from '../../helpers/pipeDuration';
+import generateCurrentDate from '../../helpers/dateGenerator';
+import { buttonText, inputText, mockedAuthorsList, mockedCoursesList } from '../../constants';
+
+export default function CreateCourse(props) {
+  const [selectedAuthorsID, setSelectedAuthorsID] = useState([]);
+  const [authorName, setAuthorName] = useState('');
+  const [duration, setDuration] = useState('');
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+
+  const handleAddAuthor = (e) => {
+    const selectedID = e.target.parentElement.getAttribute('authorid');
```

Comment:
Peeking into parent dom structure is not a good practice in general and not needed here

Why not pass author id as an argument instead?

Reply:
my bad, sorry, fixed that